### DISCUSSION TITLE
How to prevent the response from a new window?

### DISCUSSION THREAD

**Discussion by SimonasB88** [cypress-io/cypress#23286] [Questions and Help]
Writing a test for ecommerce and stuck in a PayPal payment method, where redirection goes to a new window - Cypress command lines disappear instantly. There is no `target=_blank` in a button. 

https://user-images.githubusercontent.com/29095235/184161370-829574db-27d6-4690-babc-16031cbcc604.mp4

Maybe there is a quick fix in this? Thanks.

**Comment by jjhelguero** [comment]
There might be a paypal script that redirects you to their url. You may need to intercept, wait on it, and then use .origin to handle it in cypress.

**Reply by SimonasB88** [reply]
@jjhelguero thanks, maybe you have a code snippet example?

**Comment by SimonasB88** [comment]
FYI when the new cross-origin redirection opens, the Cypress logs are showing very long requests with URLs like variables. Maybe that's the problem? How to overcome that?
![image](https://user-images.githubusercontent.com/29095235/185950455-031760af-c35f-46b4-991e-62247df1021a.png)
